updated bio for Yuning (#248)
osuosl/osuosl-pelican: content/about/about_people.rst

People

> ====== :title: Staff

Reviewer: This needs to be fixed.

.. raw:: html

  <br/>

> Yuning Gao`_

Reviewer: You're missing ` at the front.

```suggestion
`Yuning Gao`_
```

Reviewer: @Gaoyuning can you please fix this?

----------------

.. image:: /images/yuning.jpg
    :width: 110px
    :align: right
    :alt: Yuning Gao

.. class:: no-breaks

  **Media & Events Coordinator/Writer**

Yuning joined the OSL in April 2019 and is studying Psychology, Computer Science and Marketing.
She first learned about the OSL during a Marketing related job searching.
Yuning joined the lab to contribute to the open source community as well as to gain
knowledge about the social media management and event planning.

> In her free time she enjoys kendo, watch movies and camping with hammock.

Reviewer 2: Kendoooooo!!!! :crossed_swords: :exclamation:

.. raw:: html

  <br/>

`Yuning Gao`_
----------------

.. image:: /images/yuning.jpg
    :width: 110px
    :align: right
    :alt: Yuning Gao

.. class:: no-breaks

  **Media & Events Coordinator/Writer**

Yuning joined the OSL in April 2019 and is studying Psychology, Computer Science and Marketing.
She first learned about the OSL during a Marketing related job searching.
Yuning joined the lab to contribute to the open source community as well as to gain

> knowledge about the social media management and event planning.

Reviewer 2: ```suggestion
knowledge about social media management and event planning.
```

> In her free time, she enjoys Kendo, watch movies, and camping with hammocks.

Reviewer 2: ```suggestion
In her free time, she enjoys Kendo, watching movies, and camping with hammocks.
```
I forgot to suggest this in the last review. Do you think this should be in `V-ing` form?

> In her free time she enjoys kendo, watch movies and camping with hammock.

Reviewer 2: ```suggestion
In her free time, she enjoys Kendo, watch movies, and camping with hammocks.
```
Comma :policeman:

> She is also wroking in Sport Programs and various kinds of student support organizations.

Reviewer 2: "wroking" as in [this definition](https://www.urbandictionary.com/define.php?term=wroking)? Do you mean "working?

```suggestion
She is also working in Sports Programs and various kinds of student support organizations.
```
